### PR TITLE
Download latest npis when creating a release

### PR DESCRIPTION
**Pull request by jamesrhein**
Downloads the latest npis from the location given to the make target
Creates a tarball to be used to deploy to heroku

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,24 +1,28 @@
 TARGET_OS = linux
 TARGET_ARCH = amd64
-RELEASE_REPO ?= $(GOPATH)/src/github.com/adhocteam/coverage-validator-release
-RAWNPPESCSV ?= npidata_20050523-20171112.csv
-
+RELEASE_DIR = /tmp/coverage-validator-release
 SOURCES = index.html docs.html index_schema.json providers_schema.json plans_schema.json drugs_schema.json static npis.csv Procfile
+NPI_URL:=
```
Comment (gcs272) [inline Makefile]: I'm a little lost as to what this is doing - is it setting NPI_URL to a null string?

File: tools/npi-csv [hunk 2]
```diff
@@ -3,4 +3,4 @@
 set -e
 set -o pipefail
 
-awk -F, '{ printf "%s,%s\n", $1, $2 }' | tr -d '"'
\ No newline at end of file
+awk -F'\t' '{ if($2 != "\\N"){printf "%s,%s\n", $1, $2;}}' | tr -d '"'
```
Comment (gcs272) [inline tools/npi-csv]: $0.02: did it switch from csv to tsv?

**Review by gcs272 — APPROVED** [review]
still a little confused about `NPI_URL`, but lgtm

File: tools/npi-csv (hunk 2 above)
Comment (jamesrhein) [reply tools/npi-csv]: the npis-latest-dump.csv file is tab separated on s3.

File: Makefile (hunk 1 above)
Comment (jamesrhein) [reply Makefile]: You set it when you call the make target so that we aren't putting the URL in the code.

Comment (jamesrhein) [reply Makefile]: It's just the location of the npi dump file in s3

**Review by christophermanning — APPROVED** [review]

**Pull request merged**